plugins/python/src/dyncall.cpp

```
@@ -0,0 +1,129 @@
+// now the minimal dependency,  but an alternative could be based on JITing
+// using Cling or even Numba's llvmlite.
+
+#include "dyncall.hpp"
+#include <stdexcept>
+#include <string>
+
+#include <ffi.h>
+
+using namespace phlex::experimental;
+
+phlex::experimental::dcarg phlex::experimental::dcarg::from_str(std::string const& stype)
```

Cppcheck (2.21.0)

[style] The function 'from_str' is never used.

(unusedFunction)